corehq/pillows/case.py

```diff
@@ -39,15 +43,16 @@ def transform_case_for_elasticsearch(doc_dict):
 
 def get_case_to_elasticsearch_pillow(pillow_id='CaseToElasticsearchPillow', num_processes=1,
                                      process_num=0, **kwargs):
+    # todo; to remove
     assert pillow_id == 'CaseToElasticsearchPillow', 'Pillow ID is not allowed to change'
-    checkpoint = get_checkpoint_for_elasticsearch_pillow(pillow_id, CASE_INDEX_INFO, topics.CASE_TOPICS)
```

Comment:
Hey @emord Is it necessary to send the ES index date suffix to the checkpoint ID after combining, is that info useful for checkpoints at all? Asking since, after combining there might be multiple ES indices that a pillow is sending to, wondering if I should append date suffixes for all the ES indices so that checkpoint gets updated each time any ES index is updated.

Reply:
Also, note that per the [previous discussion](https://github.com/dimagi/commcare-hq/pull/21329#discussion_r205271054), in reindexer code, I have replaced the pillow with just the processor. For e.g. [see this above](https://github.com/dimagi/commcare-hq/pull/21329/files?utf8=%E2%9C%93&diff=unified#diff-84c9113bb9325ee679f1d92010e7afcaR65)

Reply:
(For latter reference). The reindexing is going to work by adding a new standalone pillow pointing to new ES index that catches up with the combined pillow (which points to old/current ES index), after which the combined pillow can be pointed to new ES index and the standalone pillow be deleted. It is easy to rollout a new standalone pillow as we have added pillows back.

Reply:
@emord I have added the index name to combined pillows anyway, since it will be easy to understand which index the combined pillow is pointing to when there is reindex